examples/v1beta1/pipelineruns/pipelinerun-with-final-results.yaml

```
@@ -0,0 +1,74 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: sum-and-multiply-pipeline
+spec:
+  params:
+    - name: a
+      type: string
+      default: "1"
+    - name: b
+      type: string
+      default: "1"
+  results:
+    - name: finally-result
+      description: "grabbing results from the finally section"
+      value: $(finally.exponent.results.product)
```

let's add a non-finally result here too 
```suggestion
    - name: task-result
      description: "grabbing results from the tasks section"
      value: $(tasks.multiply-inputs.results.product)
    - name: finally-result
      description: "grabbing results from the finally section"
      value: $(finally.exponent.results.product)
```

---

done